CRM/Contribute/BAO/Contribution.php

```diff
@@ -4186,11 +4186,13 @@ public static function getPaymentInfo($id, $component, $getTrxnInfo = FALSE, $us
             CRM_Core_Action::mask(array(CRM_Core_Permission::EDIT)),
             array(
               'id' => $resultDAO->id,
+              'contri_id' => $contributionId,
```

Comment:
Do we have to use this abbreviation - what is so long about the word contribution that we have to abbreviate it into something hard to read?